BZ-2002311: Removing stray paragraph (#36164)
openshift/openshift-docs: modules/modify-unavailable-workers.adoc

// Module included in the following assemblies:
//
// * scalability_and_performance/recommended-host-practices.adoc
// * post_installation_configuration/node-tasks.adoc

[id="modify-unavailable-workers_{context}"]
= Modifying the number of unavailable worker nodes

By default, only one machine is allowed to be unavailable when applying the kubelet-related configuration to the available worker nodes. For a large cluster, it can take a long time for the configuration change to be reflected. At any time, you can adjust the number of machines that are updating to speed up the process.

.Procedure

> . Run:

Reviewer: Maybe:
```suggestion
. To edit the `worker` machine config pool, run the following command:
```

Author: Thanks. I just copied what was in the other PR.

+
----
$ oc edit machineconfigpool worker
----

> . Set `maxUnavailable` to the value that you want:

Reviewer: Maybe:

```suggestion
. Add the `maxUnavailable` parameter and value to the spec:
```
